v4_MIGRATION_GUIDE.md

```diff
@@ -49,7 +51,8 @@ In the `AndroidManifest.xml`, migrate the `intent-filter` from your v3 integrati
 </activity>
 ``` 
 
-Note: `android:exported` is required if your app compile SDK version is API 31 (Android 12) or later.
+Additionally, apps that use both DropIn and BraintreeClient should specify a custom url scheme, since `DropInActivity` already uses the `${applicationId}.braintree` url intent filter.
```

Comment:
```suggestion
Additionally, apps that use both Drop-in and BraintreeClient should specify a custom url scheme, since `DropInActivity` already uses the `${applicationId}.braintree` url intent filter.
```